tests/integration/targets/s3_sync/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/aws
+shippable/aws/group1
+disabled
```

Comment:
I think the CI policies should allow these tests to run...

Reply:
I had a bit of difficulties finding proper documentation that defines the policies around shippable groups and when test are disabled etc.. 
Could you please point me out to the proper doc?

Reply:
The AWS specific docs:
https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/integration-aliases.html

The general ansible integration test docs:
https://docs.ansible.com/ansible/latest/dev_guide/platforms/aws_guidelines.html#integration-tests-for-aws-modules


In short:

```
cloud/aws
```
This part is mostly boilerplate, it tells CI that it's a AWS test and needs access to AWS credentials

```
shippable/aws/group1
```
This then groups the tests together, we currently have 4 'groups' of AWS tests, each of which are run against Python 2.x and 3.x, with Ansible 2.9, 2.10 and devel.  (6 copies of the test all running in parallel)

```
disabled | unsupported | unstable
```
We have 3 states that we use for tests that might not always be run:

- disabled: generally means the test is broken and needs some work, mostly caused by a change in Ansible that's not been fixed yet
- unsupported: This means that for whatever reason it can't be run in our CI environment, and is generally used for security related tests (user/group/policy management), or tests which take a long time to run.  Generally the reviewer will manually run these tests against their own environment
- unstable: This is used where tests are flaky and keep failing, usually caused by modules returning before a change has completed on the AWS side or not handling things like AWS rate-limiting errors.  These tests are only run when someone modifies that specific module.


It's also possible to add an alias to another module which is closely bound with the named module, for example the <module_name>_info companion to a module, so that you don't need to duplicate the setup / clean-up of resources, and can also help test that modules actually made the changes rather than just returning changed data.

Reply:
thanks for for the explanation and pointing me to the documentation. I've updated the aliases.